addons/l10n_in/models/account_invoice.py

```diff
@@ -61,6 +62,40 @@ def _compute_l10n_in_state_id(self):
             else:
                 move.l10n_in_state_id = False
 
+    @api.depends('state')
+    def _compute_l10n_in_tcs_tds_warning(self):
+        for move in self:
+            if move.state == 'posted':
+                warning_sections = []
+                partner_pan = move.partner_id.l10n_in_pan
+                company_pan = move.company_id.l10n_in_pan
+                per_transection_obj = self._l10n_in_calculate_per_transection_object()
+                aggregate_obj = self._l10n_in_calculate_aggregate_object(partner_pan, company_pan)
```

Comment:
```suggestion
                per_transection_obj = self._l10n_in_calculate_per_transection_object()
                aggregate_obj = self._l10n_in_calculate_aggregate_object(
                    move.partner_id.l10n_in_pan, 
                    move.company_id.l10n_in_pan
                )
```
Can't see any re usability of this variables `partner_pan` , `company_pan`